### PR TITLE
OCI-993 Refactored export_data in order for it to generate the csv an…

### PR DESCRIPTION
**Pull request by PMartins03**
…d the md files for different phases and versions.

### CODE REVIEW

File: export_data/utils/utils.go [hunk 1]
```diff
@@ -0,0 +1,211 @@
...
+		if line[2] != "" {
+			if err := writer.Write(line); err != nil {
+				log.Fatal("Failed to write to file: ", err)
+			}
+		}
+	}
+}
+
+func FilterDuplicateEntries(inputFile string, separator rune) {
+	fileRead, _ := os.Open(inputFile)
```
Comment (RossH-Raidiam) [inline export_data/utils/utils.go:166]: Same here

File: export_data/utils/utils.go [hunk 2]
```diff
@@ -0,0 +1,211 @@
...
+
+	// Read lines from file
+	reader := csv.NewReader(fileRead)
+	reader.Comma = separator
+	lines, _ := reader.ReadAll()
```
Comment (RossH-Raidiam) [inline export_data/utils/utils.go:171]: Same here

**Review by RossH-Raidiam — CHANGES_REQUESTED** [review]
Looks good overall - just avoid swallowing errors for the sake of others who may use / refactor this later down the line. Other than that, really good!

**Review by RossH-Raidiam — CHANGES_REQUESTED** [review]

File: export_data/utils/utils.go [hunk 3]
```diff
@@ -0,0 +1,229 @@
...
+func FilterEntriesWithoutConsents(inputFile string, separator rune) {
+	fileRead, err := os.Open(inputFile)
+	if err != nil {
+		log.Fatal("Failed to open file: ", err)
+	}
+
+	// Read lines from file
+	reader := csv.NewReader(fileRead)
+	reader.Comma = separator
+	lines, _ := reader.ReadAll()
```
[the remark this answers is not in the document]
Comment (PMartins03) [reply export_data/utils/utils.go:146]: Also missed it

**Review by RossH-Raidiam — APPROVED** [review]

**Pull request merged**